src/vc_sqnm/mrchem_optimizer.hpp

```diff
@@ -0,0 +1,136 @@
+#include <iostream>
+#include <string>
+
+#include <MRCPP/Timer>
+#include <MRCPP/Parallel>
+
+#include "driver.h"
+#include "mrchem.h"
+#include "mrenv.h"
+#include "version.h"
+
+#include "chemistry/Molecule.h"
+// #include "chemistry/PhysicalConstants.h"
+#include "vc_sqnm/periodic_optimizer.hpp"
+
+#include <Eigen/Dense>
+
+using json = nlohmann::json;
+using namespace mrchem;
+
+Eigen::MatrixXd getPositions(json mol_inp) {
```

Comment:
pass by const-reference because you're not modifying `mol_inp` and you avoid to take a, possibly expensive, copy
```suggestion
Eigen::MatrixXd getPositions(const json &mol_inp) {
```